arch/arm/boot/dts/overlays/README

```diff
@@ -167,6 +167,27 @@ Info:   Overlay for activation of ADAU1977 ADC codec over I2C for control
 Load:   dtoverlay=adau1977-adc
 Params: <None>
 
+Name:   ads1015
+Info:   Overlay for activation of Texas Instruments ADS1015 ADC over I2C
+Load:   dtoverlay=ads1015
+Params: addr                    I2C bus address of device. Set based how
+                                the addr pin is wired (default of 0x48
+                                assumes addr is pulled to GND)
+        chA_enable              Enable virtual channel A. (default=true)
```

Comment:
The ePAPR standard says property names should be lowercase. The tools (e.g. dtc) don't seem to mind uppercase, but I'd prefer to avoid it - sorry.